mkosi.conf.d/20-opensuse.conf

```diff
@@ -22,6 +22,7 @@ Packages=
         erofs-utils
         grep
         grub2-i386-pc
+        grub2-x86_64-efi
```

Comment:
won't this break building !x86 images?

Reply:
The opensuse image doesn't build on !x86 to begin with. I make these !x86 buildable on an as-needed basis and nobody's needed !x86 suse yet.